notebooks/summer-school/2022/lec-1.ipynb

```diff
@@ -0,0 +1,51 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9a253f80",
+   "metadata": {},
+   "source": [
+    "# History of Quantum Computing and Motivation for Quantum Simulation\n",
+    "\n",
+    "<div class=\"youtube-wrapper\">\n",
+    "    <iframe src=\"https://www.youtube.com/embed/P47zObtHsPM\" title=\"YouTube video player\" frameborder=\"0\" allow=\"accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>\n",
+    "</div>\n",
+    "\n",
+    "<div class=\"youtube-wrapper\">\n",
+    "    <iframe src=\"https://www.youtube.com/embed/zG1ZxZhMjpQ\" title=\"YouTube video player\" frameborder=\"0\" allow=\"accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>\n",
+    "</div>\n",
+    "\n",
+    "This lecture by Olivia Lanes was originally created for the 2022 Qiskit Global Summer School on Quantum Simulation.\n",
+    "\n",
+    "### Suggested reading\n",
+    "\n",
+    "- [_Introduction To Quantum and Classical Computing_](http://www.thomaswong.net/introduction-to-classical-and-quantum-computing-1e3p.pdf) by Thomas Wong\n",
+    "- [_The Strange World of Quantum Mechanics_](http://www.ba.ipcf.cnr.it/didattica/corsi/solid_state_chem/qm/The_Strange_World_of_Quantum_Mechanics_Styer.pdf) by Daniel F. Styer\n",
+    "- [_Introduction to Quantum Mechanics_](https://www.fisica.net/mecanica-quantica/Griffiths%20-%20Introduction%20to%20quantum%20mechanics.pdf) by David J. Griffiths\n",
+    "- _Learn Quantum Computation Using Qiskit:_\n",
+    "  - [_1.1 Introduction_](https://qiskit.org/textbook/ch-states/introduction.html)\n"
```

Comment:
Out of curiosity, should this link to the Textbook beta page [here](https://learn.qiskit.org/course/ch-states/introduction)?

Same question for other links in the Suggest Reading of different pages... will we eventually start routing users to the same content in Textbook beta, or for now do we still want them heading back to the old Textbook?

Reply:
Good catch, yes I think it's probably best we point to the new textbook as some of the old pages are outdated now. I've fixed this in [aa5405e](https://github.com/Qiskit/platypus/pull/1511/commits/aa5405e67a39149c79fac656f6102d4d7a0aa531), but left the link in the lab1 notebook, as I feel that's more archival than a living reference.